docs/topics/clusterdefinitions.md

```diff
@@ -75,21 +75,20 @@ $ aks-engine get-versions
 
 #### addons
 
-`addons` describes various addons configuration. It is a child property of `kubernetesConfig`. Below is a list of currently available addons:
+`addons` is an interface to define user-configurable, "base" Kubernetes componentry. It is a child property of `kubernetesConfig`. Below is a list of currently available `addons`:
```

Comment:
does "base" really make sense in this context? or I guess, what do you mean by "base"? I would call `kube-dns` base but I wouldn't call `aci-connector` or `omsagent` "base". Maybe the sentence makes more sense just as `addons is an interface to define user-configurable Kubernetes componentry`.